### PR TITLE
[tests][cdi_test]: Use exec() for ProcessTemplateWithParameters

### PR DESCRIPTION
**Pull request by myakove**

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by myakove** [comment]
ci add to whitelist

**Comment by cynepco3hahue** [comment]
ci add to whitelist

**Comment by myakove** [comment]
ci test please

**Comment by myakove** [comment]
@gouyang renamed to OcExec()

**Comment by shiywang** [comment]
@myakove thanks for the pr, I think it should be better to rename to `execute()` or `cmdExec()` or sth, because it will not be only execute OC in the future if we still want to keep this function.

others LGTM, thanks!!

**Comment by gouyang** [comment]
I rename it to `execute` in #344.

File: tests/util.go [hunk 1]
```diff
@@ -79,38 +80,32 @@ func RemoveNamespaces() {
...
 func ProcessTemplateWithParameters(srcFilePath, dstFilePath string, params ...string) string {
 	By(fmt.Sprintf("Overriding the template from %s to %s", srcFilePath, dstFilePath))
-	args := []string{"process", "-f", srcFilePath}
-	for _, v := range params {
-		args = append(args, paramFlag)
-		args = append(args, v)
-	}
-	out, err := ktests.RunOcCommand(args...)
-	Expect(err).ToNot(HaveOccurred())
+	out := execute(Result{verb: "process", filePath: srcFilePath, params: params})
```
Comment (lukas-bednar) [inline tests/util.go]: was that decision made regarding to `oc/kubectl` execution already?

File: tests/util.go [hunk 2]
```diff
@@ -137,7 +132,11 @@ func exec(r Result) {
 	if r.nameSpace != "" {
 		cmd = append(cmd, "-n", r.nameSpace)
 	}
-
+	if len(r.params) > 0 {
+		for _, v := range r.params {
+			cmd = append(cmd, "-p", v)
```
Comment (lukas-bednar) [inline tests/util.go]: there is `paramFlag` defined in contains.

File: tests/util.go (hunk 1 above)
Comment (myakove) [reply tests/util.go]: I don't know.

File: tests/util.go (hunk 2 above)
Comment (myakove) [reply tests/util.go]: Done

**Comment by gbenhaim** [comment]
@lukas-bednar Are you OK with this change?

**Pull request merged**